snuba/query/query_settings.py

```diff
@@ -79,6 +83,7 @@ def __init__(
         # TODO: is this flag still relevant?
         legacy: bool = False,
         referrer: str = "unknown",
+        asynchronous: bool = False,
```

Comment:
If you want this flag to be populated from the Request itself, you'll need to add it to here: https://github.com/getsentry/snuba/blob/master/snuba/request/schema.py#L157

Otherwise this will error. Also, this field should probably be added to the SnQL SDK request `Flags` class at some point.

Reply:
yeah, I'm working on a PR that uses the flag and I've added it in the schema there.